### PR TITLE
Working on toybox script

### PR DESCRIPTION
**Pull request by SamGondelman**
grabHands.js will let you pick up collide-able objects using any hand controller.  It can also create a table with tons of randomly placed and randomly sized toys for you to play around with.

Also updated squeeze hands to be controller agnostic but the animation doesn't seem to be working.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3256e30f02fb362081fae18bb7fd8f813ad6d92f available at: https://gist.github.com/17b6a03b45041ba29d8b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.dmg

**Comment by howard-stearns** [comment]
squeezeHands.js was working as of https://github.com/highfidelity/hifi/pull/5490
Is it broken again? How?

**Comment by SamGondelman** [comment]
@howard-stearns This is on the plugins branch so it's completely possible that a merge from master will fix the problem but if not I'll keep you updated.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 54f64b62eb3a776454e45fa80e9a173002e20a11 available at: https://gist.github.com/17b6a03b45041ba29d8b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4c7c2e93902a01313b438f75668c6743321f213d available at: https://gist.github.com/17b6a03b45041ba29d8b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5518.dmg

File: examples/controllers/handGrab.js [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    }
+}
+
+function setGrabbedObject(hand) {
+    var handPosition = (hand == LEFT) ? MyAvatar.getLeftPalmPosition() : MyAvatar.getRightPalmPosition();
+    var entities = Entities.findEntities(handPosition, GRAB_RADIUS);
+    var objectID = null;
+    var minDistance = GRAB_RADIUS;
+    for (var i = 0; i < entities.length; i++) {
+        if ((hand == LEFT && entities[i] == rightHandObjectID) ||
```
Comment (ZappoMan) [inline examples/controllers/handGrab.js]: your parentheses don't match...

Comment (SamGondelman) [inline examples/controllers/handGrab.js]: whoops, will fix as I continue updating the script

**Pull request merged**